server/controller/handler/handlers.go

```diff
@@ -35,31 +35,24 @@ import (
 // ChaosChartPath refers the location of the freshly updated repository
 var ChaosChartPath = os.Getenv("GOPATH") + "/src/github.com/litmuschaos/chaos-charts/"
 
-func checkError(err error) {
-	if err != nil {
-		log.Fatal(err)
-	}
-}
-
 func pathParser(w http.ResponseWriter, path string) {
 	var fileLookedPath = ChaosChartPath + path
 	dat, err := ioutil.ReadFile(fileLookedPath)
-	checkError(err)
+	checkIfError(err, "Path error")
```

Comment:
We will use the function like this if we are using `fatalf` but here we are not using that so we have to check error using if statement, you can take help from here https://github.com/litmuschaos/chaos-operator/blob/master/pkg/controller/chaosengine/chaosengine_controller.go